src/components/02-components/card-number/card-number.stories.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Please rename this component to `CardExperienceHours` and the file to `card-experience-hours`. `CardNumber` is a very generic name and this component will only be used in this context.